### PR TITLE
feat: Appsmith Design system V2

### LINKED ISSUE #6410: [Epic] Appsmith Design system
## Figma
https://www.figma.com/file/i80YbQLLs5mXthjIHclPpX/Appsmith-Design-System?node-id=286%3A0

### PR DESCRIPTION
**Pull request by albinAppsmith**
## Description

Entire platform theme changed to light theme

Fixes #6410

## Type of change

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- Tested UI manually
- Ran cypress test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: epic/appsmith-design-system-v2 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 54.77 **(-0.03)** | 36.86 **(-0.05)** | 33.77 **(-0.05)** | 55.38 **(-0.04)**
 :green_circle: | app/client/src/comments/CommentCard/CommentCard.tsx | 71.98 **(0.16)** | 59.06 **(0)** | 44.19 **(1.33)** | 72.35 **(0.16)**
 :red_circle: | app/client/src/comments/CommentCard/CommentContextMenu.tsx | 78.85 **(-1.54)** | 81.82 **(0)** | 30.77 **(-2.56)** | 79.59 **(-1.66)**
 :red_circle: | app/client/src/comments/CommentCard/ResolveCommentButton.tsx | 92.86 **(-0.47)** | 68.75 **(-2.68)** | 75 **(-8.33)** | 92.31 **(-0.55)**
 :green_circle: | app/client/src/components/ads/AppIcon.tsx | 40.34 **(8.57)** | 9.28 **(7.08)** | 100 **(100)** | 40.14 **(8.26)**
 :red_circle: | app/client/src/components/ads/Button.tsx | 90.85 **(-2.12)** | 73.04 **(-8.01)** | 95.56 **(0.21)** | 91.43 **(-2.22)**
 :green_circle: | app/client/src/components/ads/ButtonTabComponent.tsx | 46.67 **(3.81)** | 50 **(0)** | 0 **(0)** | 46.67 **(3.81)**
 :green_circle: | app/client/src/components/ads/Callout.tsx | 45.45 **(1.97)** | 20 **(0)** | 0 **(0)** | 45.45 **(1.97)**
 :red_circle: | app/client/src/components/ads/Checkbox.tsx | 27.03 **(0)** | 16.67 **(-2.08)** | 0 **(0)** | 30.3 **(0)**
 :green_circle: | app/client/src/components/ads/ColorPickerComponent.tsx | 37.5 **(2.5)** | 53.85 **(0)** | 0 **(0)** | 39.47 **(2.63)**
 :red_circle: | app/client/src/components/ads/DialogComponent.tsx | 88.89 **(-0.58)** | 63.64 **(0)** | 89.47 **(-1.44)** | 88.24 **(-0.65)**
 :red_circle: | app/client/src/components/ads/Dropdown.tsx | 52.9 **(-2.58)** | 36.7 **(-2.87)** | 25 **(-1.42)** | 51.06 **(-2.38)**
 :green_circle: | app/client/src/components/ads/EmojiPicker.tsx | 88.46 **(0.46)** | 90 **(-10)** | 50 **(16.67)** | 87.5 **(0.54)**
 :green_circle: | app/client/src/components/ads/EmojiReactions.tsx | 40 **(0.66)** | 25.86 **(0)** | 11.11 **(0.58)** | 40.38 **(0.76)**
 :red_circle: | app/client/src/components/ads/FilePicker.tsx | 28.92 **(-19.27)** | 9.52 **(-5.96)** | 47.83 **(-10.87)** | 30.77 **(-14.1)**
 :sparkles: :new: | **app/client/src/components/ads/FilePickerV2.tsx** | **39.84** | **11.84** | **28.57** | **34.75**
 :red_circle: | app/client/src/components/ads/Icon.tsx | 50.97 **(-3.72)** | 32.19 **(-5.02)** | 100 **(0)** | 50.84 **(-3.71)**
 :green_circle: | app/client/src/components/ads/IconTabsComponent.tsx | 36.84 **(3.51)** | 40 **(0)** | 0 **(0)** | 43.75 **(3.75)**
 :sparkles: :new: | **app/client/src/components/ads/Switcher.tsx** | **95.45** | **70.59** | **83.33** | **95**
 :red_circle: | app/client/src/components/ads/Tabs.tsx | 25 **(-1.67)** | 16 **(-0.67)** | 0 **(0)** | 26 **(-1.91)**
 :green_circle: | app/client/src/components/ads/TagInputComponent.tsx | 13.11 **(1.63)** | 5.13 **(0)** | 0 **(0)** | 14.04 **(1.76)**
 :green_circle: | app/client/src/components/ads/TextInput.tsx | 75.47 **(2.3)** | 48.81 **(3.91)** | 81.82 **(-3.89)** | 73.2 **(1.77)**
 :green_circle: | app/client/src/components/ads/Toggle.tsx | 27.27 **(2.27)** | 7.14 **(0.89)** | 0 **(0)** | 30 **(2.73)**
 :green_circle: | app/client/src/components/ads/TreeDropdown.tsx | 61.11 **(7.54)** | 58.82 **(0)** | 42.11 **(11.68)** | 59.62 **(7.77)**
 :green_circle: | app/client/src/components/designSystems/appsmith/TextInputComponent.tsx | 30.95 **(0.95)** | 17.14 **(2.33)** | 7.69 **(-1.4)** | 29.73 **(1.16)**
 :red_circle: | app/client/src/components/designSystems/appsmith/help/DocumentationSearch.tsx | 58.9 **(-0.82)** | 44.64 **(0)** | 7.14 **(-0.55)** | 60 **(-0.94)**
 :red_circle: | app/client/src/components/editorComponents/DropdownComponent.tsx | 17.02 **(-1.58)** | 12.5 **(-0.4)** | 4.17 **(-1.71)** | 20.27 **(-2.12)**
 :red_circle: | app/client/src/components/editorComponents/EditableText.tsx | 25.42 **(-1.56)** | 15.38 **(-1.64)** | 7.14 **(0.89)** | 27.78 **(-1.53)**
 :red_circle: | app/client/src/components/editorComponents/Sidebar.tsx | 92 **(-8)** | 73.33 **(-4.45)** | 70 **(-30)** | 93.62 **(-6.38)**
 :red_circle: | app/client/src/components/editorComponents/CodeEditor/styledComponents.ts | 83.93 **(-6.64)** | 63.64 **(3.17)** | 100 **(0)** | 87.04 **(-7.08)**
 :green_circle: | app/client/src/components/editorComponents/Debugger/DebuggerTabs.tsx | 48.78 **(1.28)** | 50 **(0)** | 0 **(0)** | 54.05 **(1.27)**
 :red_circle: | app/client/src/components/editorComponents/Debugger/FilterHeader.tsx | 55 **(-3.82)** | 33.33 **(0)** | 0 **(0)** | 55 **(-3.82)**
 :green_circle: | app/client/src/components/editorComponents/Debugger/LogItem.tsx | 37.68 **(1.32)** | 27.27 **(3.46)** | 0 **(0)** | 38.46 **(1.36)**
 :green_circle: | app/client/src/components/editorComponents/HighlightedCode/themes.ts | 71.43 **(4.76)** | 100 **(0)** | 0 **(0)** | 71.43 **(4.76)**
 :green_circle: | app/client/src/components/editorComponents/form/fields/EmbeddedDatasourcePathField.tsx | 30.47 **(0.55)** | 2.67 **(0)** | 4 **(0)** | 32.48 **(0.58)**
 :red_circle: | app/client/src/components/editorComponents/form/fields/KeyValueFieldArray.tsx | 50 **(-0.98)** | 29.09 **(0)** | 0 **(0)** | 51.02 **(-0.98)**
 :green_circle: | app/client/src/components/formControls/DynamicInputTextControl.tsx | 95.35 **(0.23)** | 71.43 **(4.76)** | 83.33 **(0)** | 95.24 **(0.24)**
 :green_circle: | app/client/src/components/formControls/FilePickerControl.tsx | 45.28 **(1.05)** | 40 **(0)** | 16.67 **(0)** | 46.81 **(1.16)**
 :green_circle: | app/client/src/components/formControls/StyledControls.tsx | 39.29 **(0.69)** | 82.35 **(0)** | 0 **(0)** | 39.29 **(0.69)**
 :green_circle: | app/client/src/components/propertyControls/IconSelectControl.tsx | 43.33 **(0.96)** | 26.67 **(0)** | 20 **(0)** | 45.45 **(1.01)**
 :green_circle: | app/client/src/components/propertyControls/PrimaryColumnsControl.tsx | 20.72 **(0.37)** | 12.5 **(0)** | 7.69 **(0.55)** | 22 **(0.43)**
 :red_circle: | app/client/src/components/propertyControls/StyledControls.tsx | 67.61 **(0.08)** | 75.76 **(-0.8)** | 43.9 **(-4.02)** | 67.61 **(0.08)**
 :red_circle: | app/client/src/constants/messages.ts | 74.33 **(0)** | 100 **(0)** | 23.43 **(0)** | 80.24 **(-0.29)**
 :green_circle: | app/client/src/icons/ControlIcons.tsx | 58.14 **(0.54)** | 100 **(0)** | 14.29 **(1.18)** | 58.14 **(0.54)**
 :green_circle: | app/client/src/icons/MenuIcons.tsx | 62.71 **(2.71)** | 100 **(0)** | 15.38 **(1.09)** | 62.71 **(2.71)**
 :green_circle: | app/client/src/icons/WidgetIcons.tsx | 63.01 **(1.58)** | 100 **(0)** | 20.59 **(0)** | 63.01 **(1.58)**
 :green_circle: | app/client/src/pages/Editor/EditorHeader.tsx | 87.8 **(0.08)** | 60.66 **(7.47)** | 70 **(0)** | 87.29 **(0.13)**
 :green_circle: | app/client/src/pages/Editor/EditorSaveIndicator.tsx | 86.21 **(1.02)** | 60 **(10)** | 66.67 **(0)** | 85.19 **(1.19)**
 :red_circle: | app/client/src/pages/Editor/Popper.tsx | 86.05 **(0)** | 72.5 **(0)** | 87.5 **(-1.39)** | 86.05 **(0)**
 :red_circle: | app/client/src/pages/Editor/PropertyPaneTitle.tsx | 83.33 **(0)** | 38.64 **(0)** | 81.25 **(-1.1)** | 82.35 **(0)**
 :red_circle: | app/client/src/pages/Editor/WidgetSidebar.tsx | 82.81 **(-0.31)** | 72.34 **(-2.66)** | 58.33 **(-8.34)** | 82.26 **(-0.41)**
 :red_circle: | app/client/src/pages/Editor/APIEditor/ApiRightPane.tsx | 43.1 **(-1.13)** | 50 **(2.94)** | 0 **(0)** | 47.06 **(-1.83)**
 :red_circle: | app/client/src/pages/Editor/APIEditor/Form.tsx | 37.35 **(-0.92)** | 42 **(0)** | 0 **(0)** | 38.99 **(-1.01)**
 :green_circle: | app/client/src/pages/Editor/EditorAppName/NavigationMenuItem.tsx | 25.86 **(1.79)** | 24 **(0)** | 6.25 **(-0.42)** | 27.78 **(1.78)**
 :green_circle: | app/client/src/pages/Editor/EditorAppName/index.tsx | 52.78 **(1.43)** | 26.92 **(0)** | 4 **(0.3)** | 47.62 **(1.47)**
 :red_circle: | app/client/src/pages/Editor/Explorer/EntityExplorer.tsx | 83.61 **(-0.32)** | 40.91 **(0)** | 42.86 **(-7.14)** | 84.48 **(-0.43)**
 :red_circle: | app/client/src/pages/Editor/Explorer/ExplorerIcons.tsx | 89.06 **(0.35)** | 50 **(-8.33)** | 20 **(0)** | 88.71 **(0.38)**
 :green_circle: | app/client/src/pages/Editor/Explorer/JSDependencies.tsx | 84.85 **(0)** | 100 **(6.25)** | 50 **(0)** | 89.29 **(0)**
 :green_circle: | app/client/src/pages/Editor/Explorer/index.tsx | 94.74 **(7.78)** | 66.67 **(16.67)** | 100 **(0)** | 94.74 **(7.78)**
 :green_circle: | app/client/src/pages/Editor/Explorer/Actions/MoreActionsMenu.tsx | 33.33 **(1.75)** | 33.33 **(0)** | 0 **(0)** | 36.11 **(1.82)**
 :green_circle: | app/client/src/pages/Editor/Explorer/Datasources/DatasourceStructure.tsx | 46.88 **(1.72)** | 50 **(0)** | 0 **(0)** | 51.72 **(1.72)**
 :red_circle: | app/client/src/pages/Editor/Explorer/Entity/Collapse.tsx | 88.89 **(-4.44)** | 75 **(-8.33)** | 100 **(0)** | 100 **(0)**
 :red_circle: | app/client/src/pages/Editor/Explorer/Entity/CollapseToggle.tsx | 91.67 **(-0.64)** | 75 **(-8.33)** | 100 **(0)** | 100 **(0)**
 :red_circle: | app/client/src/pages/Editor/Explorer/Onboarding/index.tsx | 44.83 **(-1.32)** | 50 **(0)** | 0 **(0)** | 48.15 **(-1.85)**
 :green_circle: | app/client/src/pages/Editor/Explorer/Pages/PageGroup.tsx | 79.49 **(0)** | 80 **(20)** | 50 **(0)** | 81.58 **(0)**
 :green_circle: | app/client/src/pages/Editor/FirstTimeUserOnboarding/Statusbar.tsx | 89.36 **(0.23)** | 77.46 **(-0.8)** | 85.71 **(1.09)** | 89.13 **(0.24)**
 :green_circle: | app/client/src/pages/Editor/PagesEditor/ContextMenu.tsx | 64.71 **(4.71)** | 60 **(0)** | 0 **(0)** | 68.75 **(5.11)**
 :green_circle: | app/client/src/pages/Editor/PagesEditor/PageListItem.tsx | 62.86 **(4.53)** | 57.14 **(1.58)** | 0 **(0)** | 62.86 **(4.53)**
 :green_circle: | app/client/src/pages/Editor/PropertyPane/PanelPropertiesEditor.tsx | 20.51 **(2.09)** | 0 **(0)** | 0 **(0)** | 21.05 **(2.13)**
 :red_circle: | app/client/src/pages/Editor/PropertyPane/PropertyPaneConnections.tsx | 65.96 **(0)** | 30.91 **(0)** | 45.45 **(-1.61)** | 65.59 **(0)**
 :green_circle: | app/client/src/pages/Editor/PropertyPane/index.tsx | 83.33 **(0.14)** | 69.3 **(0)** | 70.37 **(0)** | 85.05 **(0.14)**
 :green_circle: | app/client/src/pages/common/ProfileDropdown.tsx | 70.37 **(2.37)** | 80 **(5)** | 0 **(0)** | 70.37 **(2.37)**
 :red_circle: | app/client/src/pages/common/ProfileImage.tsx | 90 **(0)** | 68.75 **(0)** | 66.67 **(-4.76)** | 88.89 **(0)**
 :sparkles: :new: | **app/client/src/pages/common/SharedUserList.tsx** | **89.47** | **50** | **50** | **88.89**
 :green_circle: | app/client/src/pages/organization/General.tsx | 77.78 **(3.31)** | 69.23 **(-3.5)** | 33.33 **(13.33)** | 77.78 **(3.31)**
 :green_circle: | app/client/src/reducers/uiReducers/onBoardingReducer.ts | 26.92 **(3.84)** | 100 **(0)** | 5 **(5)** | 26.92 **(3.84)**
 :red_circle: | app/client/src/reducers/uiReducers/themeReducer.ts | 41.18 **(-5.49)** | 0 **(0)** | 0 **(0)** | 41.18 **(-5.49)**
 :red_circle: | app/client/src/utils/helpers.tsx | 58.44 **(-6.49)** | 28.57 **(-8.32)** | 40.54 **(-2.32)** | 55.79 **(-6.26)**
 :x: | ~~app/client/src/components/designSystems/appsmith/header/ThreeDotsLoading.tsx~~ | ~~80~~ | ~~100~~ | ~~0~~ | ~~80~~</details>

### CODE REVIEW

**Review by rishabhsaxena — APPROVED** [review]
LGTM

File: app/client/src/pages/organization/Members.tsx [hunk 1]
```diff
@@ -196,33 +283,86 @@ export default function MemberSettings(props: PageProps) {
       },
     },
   ];
-
   const currentOrgName = currentOrg?.name ?? "";
+  const isMobile: boolean = useMediaQuery({ maxWidth: 767 });
```
Comment (rishabhsaxena) [inline app/client/src/pages/organization/Members.tsx:287]: nit: should move the breakpoint widths to constants

**Review by rishabhsaxena — DISMISSED** [review]

**Comment by rishabhsaxena** [comment]
image

one minor point: we could add a border to the comments container

**Review by rishabhsaxena — APPROVED** [review]

File: app/client/src/components/ads/FilePickerV2.tsx [hunk 2]
```diff
@@ -0,0 +1,316 @@
+import React, { useEffect, useRef, useState } from "react";
```
[the remark this answers is not in the document]
Comment (albinAppsmith) [reply app/client/src/components/ads/FilePickerV2.tsx:1]: This was done very earlier thinking we may need to switch between new UI and old UI. We will merge this.

**Review by sharat87 — DISMISSED** [review]

**Review by rishabhsaxena — APPROVED** [review]

**Review by rishabhsaxena — DISMISSED** [review]

**Review by rishabhsaxena — APPROVED** [review]
approving for tests

**Review by jsartisan — DISMISSED** [review]

**Review by jsartisan — APPROVED** [review]

**Review by jsartisan — APPROVED** [review]

**Review by jsartisan — APPROVED** [review]

**Review by sbalaji1192 — APPROVED** [review]
Approving to run tests

**Review by sbalaji1192 — DISMISSED** [review]
Approving to run tests

**Review by ayushpahwa — APPROVED** [review]
Approving to start the tests.

**Review by sbalaji1192 — DISMISSED** [review]
Approving to run tests.

**Review by sbalaji1192 — APPROVED** [review]
Approving to run tests

**Review by rishabhsaxena — APPROVED** [review]
Approving for tests

**Pull request closed without merging**